### ISSUE TITLE
Multiple GPU bug

### ISSUE THREAD

**Issue by yanzq95** [alibaba/UniFuse-Unidirectional-Fusion#12]
When I set --gpu_devices as [0, 1, 2, 3], the training process can only work on single GPU. If any other setting should be conducted? Thanks!

**Comment by hualie** [comment]
The code only supports single GPU training.

**Comment by Camuvingian** [comment]
@hualie is there any fundamental reason why the training model cannot be made multi-GPU? anonymoustbd, have you made any progress with a multi-GPU training model?

**Comment by HalleyJiang** [comment]
Hi, all. Please refer to my updated repository (https://github.com/HalleyJiang/UniFuse-Unidirectional-Fusion) for multi-GPU training. --batch_size should be set as the size for each GPU. My implementation refers to https://github.com/dougsouza/pytorch-sync-batchnorm-example.
